pages/core/features/subscription.mdx

```diff
@@ -12,7 +12,7 @@ The [Subscription](https://app.starknet.id/subscription) feature makes renewing
 
 **2️⃣ Select the domain to renew & click on "enable subscription"**
 
-![img](public/illustrations/domain-renew-enable.webp)
```

Comment:
No please keep the same name as before. Just delete the former one and import the new one with the same name.